testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/rx/rxjava2/resource/Rx2FlowableableSSECompatibilityResource.java

```diff
@@ -18,13 +17,11 @@
 
    @GET
    @Path("eventStream/thing")
-   @Produces("text/event-stream")
-   @SseElementType("application/json")
+   @Produces("text/event-stream;element-type=application/json")
    void eventStreamThing(@Context SseEventSink eventSink, @Context Sse sse);
 
    @GET
    @Path("flowable/thing")
-   @Produces("text/event-stream")
-   @SseElementType("application/json")
+   @Produces("text/event-stream;element-type=application/json")
```

Comment:
I'm wondering if it would be better to have both ways of using annotations here, instead of changing all resource methods you could change just one of them and add a comment explaining the reason / referring to the JIRA number.

Reply:
Rx2ObservableSSECompatibilityTest is the way to use the @SseElementType("application/json") and we don't have to use @SseElementType here.